### PR TITLE
Seekbar more metrics

### PR DESCRIPTION
**Pull request by VolkerGoeschl**
This allows showing different metrics in the searchbar for spotting spots of interrest in longer logs easier.
The dropdown to select those is right beside the workspace selection. The chosen mode will survive picking a different log from a file but will get reset to average throttle when loading a new file.
The following modes have been implemented (so far)
Average motor throttle: this is the old behavior
maximum Stick input: highest absolute value of (roll/pitch/yaw) (find snap-moves)
maximum Motor differential: difference of highest and lowest motor output (find fast moves/desyncs - thanks to Zoggbarr for the suggestion)

ref: #634

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: js/flightlog_index.js
Comment (haslinghuis) [inline js/flightlog_index.js:103]:
```suggestion
                for (let j = 0; j < 3; j++) {
```

File: js/flightlog_index.js
Comment (haslinghuis) [inline js/flightlog_index.js:157]:
```suggestion
                                        for (let j = 0; j < maxRCFields.length; j++) {
```

File: js/flightlog_index.js
Comment (haslinghuis) [inline js/flightlog_index.js:294]:
```suggestion
                for (let j = 0; j < sourceIndex.maxRC.length; j++) {
```

File: js/flightlog_index.js
Comment (haslinghuis) [inline js/flightlog_index.js:299]:
```suggestion
                for (let j = 0; j < sourceIndex.maxMotorDiff.length; j++) {
```

File: js/main.js
Comment (haslinghuis) [inline js/main.js:339]:
```suggestion
            let
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Please fix lexical scoping and code smells

**Comment by haslinghuis** [comment]
More code smells to solve :yum:

**Review by haslinghuis — APPROVED** [review]

**Comment by VolkerGoeschl** [comment]
Now it seems to be accepted by SonarCloud.
Do I need to squash or rebase to get rid of al those micro-commits?
@haslinghuis thank you for your patience!

**Comment by haslinghuis** [comment]
We have squash and merge, so all good.

**Comment by nerdCopter** [comment]
the default window size at launch always pushes this off-view for me.
image

thinking the entire bar needs some auto-flow/sizing work :(  

thoughts @haslinghuis ??

**Comment by RubberQuads** [comment]
> the default window size at launch always pushes this off-view for me. image
> 
> thinking the entire bar needs some auto-flow/sizing work :(
> 
> thoughts @haslinghuis ??

I want to put it into the seekbar like it's done in the spectrum analyzer. But I need to get a grasp of how the canvases work before -> can we leave it as is for now and I change it to something better in another PR next week (I'm on vacation right now)?

**Comment by nerdCopter** [comment]
``[moved selector into the seekbar](https://github.com/betaflight/blackbox-log-viewer/pull/635/commits/2232583ac416fb4832452a71fcbf2b1100446def)`` is very sleek :heart: 

for others to review:
invisible until mouse over 
image

mouse-over:
image

upon modifiy:
image

**Comment by nerdCopter** [comment]
please mark any of the above suggestions resolved if they are indeed resolved.
image

File: js/flightlog_index.js
Comment (VolkerGoeschl) [reply js/flightlog_index.js]: done

File: js/flightlog_index.js
Comment (VolkerGoeschl) [reply js/flightlog_index.js]: done

**Review by nerdCopter — APPROVED** [review]

File: js/main.js
Comment (haslinghuis) [inline js/main.js:319]: Slipped through Sonarcloud bot :)

**Pull request merged**
